vpr/src/draw/buttons.cpp

```diff
@@ -26,6 +27,43 @@ gint label_left_start_col = 0;
 gint box_left_start_col = 0;
 gint button_row = 2; // 2 is the row num of the window button in main.ui, add buttons starting from this row
 
+void button_for_net_alpha() {
+    GObject* main_window = application.get_object(application.get_main_window_id().c_str());
+    GObject* main_window_grid = application.get_object("InnerGrid");
+    //grid
+    GtkWidget* grid = gtk_grid_new();
+    gtk_grid_insert_row((GtkGrid*)grid, 0);
+    gtk_grid_insert_column((GtkGrid*)grid, 0);
+
+    //text entry for apha value
+    GtkWidget* entry = gtk_entry_new();
+    std::string initialValue = std::to_string(get_net_alpha());
+    gtk_entry_set_text((GtkEntry*)entry, initialValue.c_str());
+    gtk_entry_set_input_purpose((GtkEntry*)entry, GTK_INPUT_PURPOSE_NUMBER);
+    gtk_widget_set_name(entry, "alphaValue");
+
+    //label
+    GtkWidget* alpha_label = gtk_label_new("Set alpha value (<1)");
+
+    //button
+    GtkWidget* button = gtk_button_new_with_label("set");
```

Comment:
It seems like the user must press the 'set' button to change the alpha value. It doesn't seem to work if the user enters a value and just presses enter. 

It would be good to support pressing enter.